pyannote/audio/features/utils.py

```diff
@@ -36,11 +36,7 @@
 from pyannote.core import SlidingWindow, SlidingWindowFeature
 import tempfile
 
-import scipy.io.wavfile
-import warnings
-from scipy.io.wavfile import WavFileWarning
-
-
+from soundfile import SoundFile
```

Comment:
This dependency is missing in `requirements.txt`.

Reply:
Fixed in 79b54d7 commit